services/ui-src/src/components/tables/Table.tsx

```diff
@@ -1,62 +1,76 @@
 // components
-import { Table, Tbody, Td, Th, Thead, Tr } from "@chakra-ui/react";
+import { Table as TableRoot, Tbody, Td, Th, Thead, Tr } from "@chakra-ui/react";
 // utils
 import { makeMediaQueryClasses } from "../../utils/useBreakpoint";
-import { JsonObject } from "utils/types/types";
+import { TableContentShape } from "utils/types/types";
 
-export const DueDateTable = ({ verbiage }: Props) => {
+export const Table = ({
+  content,
+  variant,
+  lastCellsBold,
+  dataTestId,
+}: Props) => {
   const mqClasses = makeMediaQueryClasses();
 
   return (
-    <Table variant="striped" size="sm" data-testid="due-date-table">
+    <TableRoot variant={variant} size="sm" data-testid={dataTestId || "table"}>
       <Thead>
         <Tr>
-          {/* Table Header Row */}
-          {verbiage.header.map((headerCell: string, index: number) => (
+          {/* Head Row */}
+          {content.headRow.map((headerCell: string, index: number) => (
             <Th key={index} sx={sx.tableHeader} className={mqClasses}>
               {headerCell}
             </Th>
           ))}
         </Tr>
       </Thead>
       <Tbody>
-        {/* Table Body Rows */}
-        {verbiage.rows.map((row: string[], index: number) => (
+        {/* Body Rows */}
+        {content.bodyRows.map((row: string[], index: number) => (
           <Tr key={index}>
             {/* Row Cells */}
             {row.map((cell: string, index: number) => (
-              <Td key={index} sx={sx.tableCell} className={mqClasses}>
+              <Td
+                key={index}
+                sx={sx.tableCell}
+                className={`${mqClasses}${
+                  lastCellsBold ? " boldLastCell" : ""
+                }`}
```

Comment:
Might be a cleaner way to do this. I know there are some libs that make it easier. We could also make a little utility that does it a lot less expensively.